rpmalloc/rpmalloc.c

```diff
@@ -3590,6 +3590,18 @@ rpmalloc_heap_thread_set_current(rpmalloc_heap_t* heap) {
 	}
 }
 
+extern inline rpmalloc_heap_t*
+rpmalloc_get_heap_for_ptr(void* ptr)
+{
+	//Grab the span, and then the heap from the span
+	span_t* span = (span_t*)((uintptr_t)ptr & get_memory_span_mask);
```

Comment:
and `get_memory_span_mask` should be `_memory_span_mask` - assuming another divergence :)

Reply:
Yes; and actually made sure it compiled this time :)